content/components/underline-nav.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Underline nav
+description: The UnderlineNav is used to display primary navigation.
+componentId: underline_nav
+reactId: underline_nav
+figma: https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=18843-67449&t=XqqSolC6AzRO9nae-11
+rails: https://primer.style/view-components/components/alpha/underlinenav/
+---
+
+# Underline nav
+
+The underline nav is used to provide navigation links to related views for the user's current context. It is typically used as primary navigation at the top of the page or as secondary navigation when nested in views with a [nav list](/components/nav-list).
+
+## Anatomy
+
+![Anatomy of the UnderlineNav component](https://user-images.githubusercontent.com/175638/186432408-c2479dd2-31fe-4261-9f9a-e67d49b28e72.png)
+
+1. **Item**: A navigation link.
+2. **Current item**: The current selection item is highlighted with an underline.
+3. **Label**: Describes the navigation item.
+4. **Leading icon (optional)**: An icon that identifies the navigation item.
+5. **Counter (optional)**: Displays a count next to the label.
+6. **Overflow menu**: A disclosure menu that displays additional items when there are too many to fit in the available space.
+
+## Content
+
+### Item
+
+The underline nav is composed of a list of items. Each item is a link that can be selected to navigate to a different page in the current context.
+
+#### Label text
+
+The label indicates the destination page of the link. Keep labels clear, short, and concise.
```

Comment:
"destination page" makes it sound like the user will leave their current context when clicking the link.

Maybe something like this?

```suggestion
The title of the view associated with that tab. Keep labels clear, short, and concise.
```